src/04-modality-specific-files/10-functional-near-infrared-spectroscopy.md

```diff
@@ -0,0 +1,367 @@
+64 Ch Standard-2”` and `NIRSPlacementScheme = ”n/a”` should be used.
+If an EasyCap was used but with custom positions,
+as may be done by cutting custom holes in the cap,
+then the values `CapManufacturer = ”EasyCap”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”` should be used.
+If a completely custom cap was knitted then
+`CapManufacturer = ”custom”`, `CapManufacturersModelName = ”custom”` and
+`NIRSPlacementScheme = ”n/a”`.
+If no cap was used and optodes were taped to the scalp
+at positions Cz, C1 and C2, then the values `CapManufacturer = ”none”`, `CapManufacturersModelName
+= ”n/a”` and `NIRSPlacementScheme = ”["Cz", "C1", “C2]”` should be used.
+
+Closely spaced source detector pairs are often included in fNIRS measurements to
+obtain a measure of systemic, rather than neural, activity. These source-detector
+pairs are referred to as short channels. There is variation in how manufacturers
+implement these short channels, some use specialised sources or detectors,
+and the placement mechanisms vary. The exact specifications on the type of
+specialised short channel optodes is beyond the scope of the BIDS specification
+and this information can be stored within the SNIRF file (for example, in the `sourcePower` field).
+However, to improve searchability and ease of access for users it is useful to
+know if short channels were included in the fNIRS measurements, this information
+is stored in the field `ShortChannelCount`.
+If the field `ShortChannelCount` is populated, then the optional column `short_channel`
+may be used in `*_channels.tsv` to describe which channels were specified as short.
+
+Generic fields that MUST be present:
+
+| **Key name** | **Requirement level** | **Data type** | **Description**                                                                                                                                                                                                                                                                                                                                                                    |
+| ------------ | --------------------- | ------------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| TaskName     | REQUIRED              | [string][]    | Name of the task. No two tasks should have the same name. The task label included in the file name is derived from this TaskName field by removing all non-alphanumeric (`[a-zA-Z0-9]`) characters. For example `TaskName` `"faces n-back"` will correspond to task label `facesnback`. A RECOMMENDED convention is to name resting state task using labels beginning with `rest`. |
```

Comment:
Why can't two tasks have the same name? What if they were the same task? Does this mean that two separate tasks within the same file can't have the same name? I think another sentence is needed to clarify what you mean.

Reply:
You cant have two seperate tasks in one file. I think the common understanding of task and the definition differ here, causing this confusion. I paste the definition from BIDS below.

Task - a set of structured activities performed by the participant. Tasks are usually accompanied by stimuli and responses, and can greatly vary in complexity. For the purpose of this specification we consider the so-called "resting state" a task. In the context of brain scanning, a task is always tied to one data acquisition. Therefore, even if during one acquisition the subject performed multiple conceptually different behaviors (with different sets of instructions) they will be considered one (combined) task.